app/interactors/schedule/schedule_interactor.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class Schedule::ScheduleInteractor
+  include Interactor
+  delegate :params,
+           :user,
+           :edition,
+           :issues,
+           to: :context
+
+  def call
+    Edition.transaction do
+      find_and_lock_edition
+      check_for_issues
+      schedule_publishing
+      create_scheduling
```

Comment:
I'd rather not combine them because they are doing different things, however I agree the naming is not clear.  Perhaps we could have `set_scheduled_publishing_job` instead of `schedule_publishing`?

Reply:
I think it's actually the `create_scheduling` one that was puzzling me, as 'scheduling' isn't a noun, I'm not sure how we could create one. Looking at the methods, I'm now wondering what they do that's different - they both look like they're doing scheduling, but in different ways (one is a job, one is a service; they both say schedule). I reckon this might take a bit of time to come to a conclusion, so I'll note it down on the card to think about later.